src/app/model-invoke-run/model-invoke-run.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="form-container">
+  <form [formGroup]="myForm" (ngSubmit)="onSubmit()">
+    <div class="form-group">
+      <label for="runName">Name of Model Run:</label>
+      <input id="runName" formControlName="runName" type="input">
+      @if (myForm.get('runName')!.invalid && myForm.get('runName')!.touched) {
+      <div class="error-message">
+        String field is required.
+      </div>
+      }
+    </div>
+
+    <div class="form-group">
+      <label for="numScenarios">Number of scenarios:</label>
+      <input id="numScenarios" formControlName="numScenarios" type="input">
+      @if (myForm.get('numScenarios')!.invalid && myForm.get('numScenarios')!.touched) {
+      <div class="error-message">
+        Must be a power of two.
+      </div>
+      }
+    </div>
+    <div class="form-group">
+      <label>Tabular Acropora Seeded Bounds:</label>
```

Comment:
I find this awkwardly phrased.

How about "Seeded Tabular Acropora"?

It's clear it's the bounds by virtue of the control being a range slider.